README.md

```diff
@@ -1,70 +1,52 @@
-# Azure SDK for Javascript
+# Azure SDK for JavaScript
+
 
 | Component            | Build Status                                                                                                                                                                                |
 | -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Management Libraries | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/138?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=138&branchName=master) |
 | Client Libraries     | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/45?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=45&branchName=master)   |
 
-This project provides an isomorphic Javascript package with TypeScript definitions that makes it easy to consume and manage
-Microsoft Azure Services.
-It supports SDKs for:
-
-- ARM services (control plane) (packages with the naming convention of `@azure/arm-<servicename>`)
-- data plane of some Azure services (packages with the naming convention of `@azure/<servicename>`).
-
-## Documentation
-
-Documentation of the supported SDKs can be found here:
-
-- https://docs.microsoft.com/en-us/javascript/azure - This website primarily provides SDK documentation for
-
-  - ARM based services (`@azure/arm-<serviceName>`)
-  - data plane SDKs like `@azure/batch`, `@azure/graph`, etc.
+This repository contains official JavaScript libraries for Azure services. For documentation go to [Azure SDK for JavaScript documentation](https://aka.ms/js-docs).
 
-## Authentication
+You can find a complete list of all the packages for these libraries [here](https://github.com/Azure/azure-sdk-for-js/blob/master/packages.md).
```

Comment:
"packages for these libraries" ?

Also, avoid having link texts like "here" which have no contextual information on their own.
It hurts the screenreader users when they engage the option of listing all links on the page

Reply:
"packages for these libraries" can you say more?

Agree with "here" but all the other readmes have it so I didn't fix it. I guess I will since you brought it up too :)

Reply:
From an outside in perspective, I don't see the difference between "package" and "library"